### PR TITLE
Add description to ASLayout

### PR DESCRIPTION
**Pull request by levi**
Making debugging easier.

### CODE REVIEW

File: AsyncDisplayKit/Layout/ASLayout.mm [hunk 1]
```diff
@@ -138,6 +138,12 @@ + (instancetype)layoutWithLayout:(ASLayout *)layout position:(CGPoint)position
                                sublayouts:layout.sublayouts];
 }
 
+- (NSString *)description
+{
+    return [NSString stringWithFormat:@"<<ASLayout: %p>, position = %@; size = %@; constrainedSize = %@",
+            self, NSStringFromCGPoint(self.position), NSStringFromCGSize(self.size), NSStringFromCGSize(self.constrainedSize)];
```
Comment (Adlai-Holler) [inline AsyncDisplayKit/Layout/ASLayout.mm]: Let's enclose the entire string in angle brackets like `<ASLayout: %p; position = %@; size = %@; constrainedSize = %@>`

**Comment by Adlai-Holler** [comment]
Nice! Build failures may be resolved by explicitly #importing Foundation.h or UIKit.h

File: AsyncDisplayKit/Layout/ASLayout.mm (hunk 1 above)
Comment (levi) [inline AsyncDisplayKit/Layout/ASLayout.mm]: @Adlai-Holler I was trying to make it look like a UIView description. Do you have a strong preference here?

Comment (Adlai-Holler) [inline AsyncDisplayKit/Layout/ASLayout.mm]: I believe UIView descriptions are formatted with just one set of angle brackets.

**Comment by appleguy** [comment]
Great!  @levi @Adlai-Holler we should definitely add more of these throughout Pinterest app code too.

**Pull request merged**